modules/tensor_mechanics/include/kernels/lagrangian/TotalLagrangianStressDivergence.h

```diff
@@ -0,0 +1,88 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "LagrangianStressDivergenceBase.h"
+
+/// Enforce equilibrium with a total Lagrangian formulation
+///
+/// This class enforces equilibrium when used in conjunction with
+/// the corresponding strain calculator (CalculateStrainLagrangianKernel)
+/// and with either a stress calculator that provides the
+/// 1st PK stress ("pk1_stress") and the derivative of the 1st PK stress
+/// with respect to the deformation gradient ("pk1_jacobian")
+///
+/// This kernel should be used with the new "ComputeLagrangianStressBase"
+/// stress update system and the "ComputeLagrangianStrain" system for strains.
+///
```

Comment:
I thought triple slashes `///` are for single line comments. Did you check whether this looks fine in doxygen?

Reply:
That's a valid option for a detailed class [description](https://www.doxygen.nl/manual/docblocks.html).  It looks okay in the rendered Doxygen html.  As long as I remembered to include the blank line the first line ends up as the short description and the rest is the detailed description.